wai-logger/Network/Wai/Logger.hs

```diff
@@ -44,6 +44,7 @@ module Network.Wai.Logger (
   , initLogger
   -- * Types
   , IPAddrSource(..)
+  , LoggedPathType(..)
```

Comment:
I don't think `LoggedPathType` is necessary.
Just change the behavior to record path+query-string rather than path only.
With this approach,  we don't have to change other packages and CI will be fixed.